content/topical/flash_floods.md

```diff
@@ -7,10 +7,12 @@ work_package: 4
 license: CC-BY
 format: NetCDF
 size: 100 GB
-doi:
+doi: TBA
```

Comment:
Rather keep it empty (we want it to itch a bit)

Reply:
Sure. :)